### PR TITLE
[pytest] Fix the fast failure when checking the format of Monit alerting message

### PR DESCRIPTION
**Pull request by yozhao101**
Signed-off-by: Yong Zhao <yozhao@microsoft.com>

### Description of PR

Summary:
Fixes # (issue)

### Type of change

- [x ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ x] 201911

#### What is the motivation for this PR?
This PR aims to fix the fast failure issue when checking the format of Monit alerting message. Initially `check_monit_last_output(...)` was called by `wait_until(...)` to continuously check whether the format of Monit alerting message was correct or not. If the Monit alerting message was not found, this script should return false instead of failing this test directly.

#### How did you do it?
I replaced the statement `pytest.fail(...)` in the function `check_monit_last_output(...)` with `return False`. At the same time, some logging statements are added in order to show what action was done during testing.

#### How did you verify/test it?
I test this change on the DuT `str-msn2700-22`.

#### Any platform specific information?
N/A

#### Supported testbed topology if it's a new test case?
N/A

### CODE REVIEW

File: tests/monit/test_monit_status.py [hunk 1]
```diff
@@ -31,13 +31,34 @@ def disable_lldp(duthosts, enum_rand_one_per_hwsku_frontend_hostname):
         None.
     """
     duthost = duthosts[enum_rand_one_per_hwsku_frontend_hostname]
-    duthost.command("docker exec lldp supervisorctl stop lldpmgrd")
+
+    logger.info("Stopping 'lldpmgrd' process in 'lldp' container ...")
```
Comment (bingwang-ms) [inline tests/monit/test_monit_status.py:35]: Just a double confirm, do we need to run this on multi_asic DUT? I don't have a multi_asic device to verify.

File: tests/monit/test_monit_status.py [hunk 2]
```diff
@@ -31,13 +31,34 @@ def disable_lldp(duthosts, enum_rand_one_per_hwsku_frontend_hostname):
...
+        logger.info("Stopping 'lldpmgrd' process in 'lldp0' container ...")
+        stop_command_result = duthost.command("docker exec lldp0 supervisorctl stop lldpmgrd")
+        exit_code = stop_command_result["rc"]
+        pytest_assert(exit_code == 0, "Failed to stop 'lldpmgrd' process in 'lldp0' container!")
+        logger.info("'lldpmgrd' process in 'lldp0' container is stopped.")
+
     yield
-    duthost.command("docker exec lldp supervisorctl start lldpmgrd")
+
+    logger.info("Starting 'lldpmgrd' process in 'lldp' container ...")
```
Comment (bingwang-ms) [inline tests/monit/test_monit_status.py:50]: Same above.

**Comment by bingwang-ms** [comment]
Other update looks good to me. Please help to confirm if Line 34 ~ 39 is needed on multi_asic DUT.

File: tests/monit/test_monit_status.py (hunk 1 above)
Comment (yozhao101) [reply tests/monit/test_monit_status.py:35]: Yes, I think we need stop `lldpmgrd` process in `lldp` container which is running on Host and on each ASIC to check the format of Monit alerting message. Let me test this change on Multi-ASIC device again to verify.

Comment (yozhao101) [reply tests/monit/test_monit_status.py:35]: Bing, I tested this change on DuT `str-n3164-acs-2` again and it passed.

**Review by bingwang-ms — APPROVED** [review]

**Pull request merged**